frontends/android/BitBoxApp/app/src/main/java/ch/shiftcrypto/bitboxapp/MainActivity.java

```diff
@@ -245,12 +247,12 @@ protected void onResume() {
         // DETACHED intent is a broadcast intent which we register here.
         IntentFilter filter = new IntentFilter();
         filter.addAction(UsbManager.ACTION_USB_DEVICE_DETACHED);
+        filter.addAction("ch.shiftcrypto.bitboxapp.USB_PERMISSION");
```

Comment:
Not sure why this commit adds the action here. It doesn't seem to be handled in `usbStateReceiver` as per docs:

```java
private static final String ACTION_USB_PERMISSION =
    "com.android.example.USB_PERMISSION";

    public void onReceive(Context context, Intent intent) {
        String action = intent.getAction();
        if (ACTION_USB_PERMISSION.equals(action)) {
          ...
        }
   }
```

I assume it'll be used in a following commit but seems misplaced here.

Reply:
It is handled in `handleIntent()` (in the same commit) which is called from the usb state receiver.